auth/jwt.go

```diff
@@ -111,7 +111,8 @@ func (id *jwtIdentity) Valid() error {
 	}
 
 	now := jwt.TimeFunc().UTC().Unix()
-	if now < id.IssuedAt {
+	// provide tolerance for clockdrifting slower
+	if now < id.IssuedAt-int64(ClockDriftDelta.Seconds()) {
```

Comment:
Answering the first question:  Yes, we pass this object as a jwt.Claims, which requires a "Valid" method.